docs/changes/io.ascii/13130.feature.rst

```diff
@@ -0,0 +1,3 @@
+Added support in ``io.ascii`` for reading and writing file paths of the form
+``~/file.csv`` or ``~username/file.csv``, where the ``~`` prefix indicates the
```

Comment:
`~username` does not quite make sense to me. 
```suggestion
``~/file.csv`` or ``/home/username/file.csv``, where the ``~`` prefix indicates the
```

Reply:
Maybe it should be `~<username>/file.csv`? It's a less common syntax for referring to other accounts' home directories---logged into my account, `~` resolves to `/home/svank` and `~pllim` resolves to `/home/pllim`.

Both are supported by `os.path.expanduser`, but I suspect very few people will ever use the `~<username>` form. I also wasn't quite sure how to describe this concisely in the changelog, especially since any concrete example assumes an operating system.

Reply:
Interesting... I never thought to type the extra `username` when `~` is enough. Thanks for the clarification! 😸

Reply:
No problem, I've re-worded the changelog entry to try to make it more clear

Reply:
I'd hardly use the explicit `username` for my own home dir, but directing on other's directories on real multi-user systems I am using the `~<username>/` syntax quite regularly, so +1 for pointing that out as it's now done.

Reply:
I use `~username` for referring to other people's usernames. I;ve never seen it with `<>` but I'm glad I have the chance to learn something new!

Reply:
I take it the `<>` are only added to point out it's a placeholder for the actual username (as in "`~<username>` is equivalent to `$<USERNAME>`") – looks clearer to me that way, but if that can cause confusion, maybe we should reconsider (I'd expect actual `<`, `>` to wreak all kind of redirection havoc!).

Reply:
It seems that both forms can cause confusion as I was clearly confused 8 min ago. BUT - we are discussing the changelog of astropy here. This is not even astropy functionality, it's only about passing through to `os.path.expanduser` and, as @svank pointed out it's actually OS dependent. We are not writing a unix manual here. So, I think either form is fine i.e. I suggest to leave it as is and direct our energy to other open PRs that do need work.